src/System.IO.Packaging/src/System/IO/Packaging/PackagePartCollection.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+//-----------------------------------------------------------------------------
+//
+// Description:
+//  This is a base abstract class for PackagePartCollection. This is a part of the 
+//  MMCF Packaging Layer
+//
+//-----------------------------------------------------------------------------
+
+// Allow use of presharp warning numbers [6506] unknown to the compiler
```

Comment:
Removed all comments referencing presharp, also removed pragmas related to presharp.